community/README.md

```diff
@@ -1,65 +1,62 @@
-## Nebula Graph Community Architecture
+# Nebula Graph Community Architecture
 
 The Nebula Graph community is organized as shown below.
 
 ![Nebula Community Architecture](./images/structure.png)
 
-### PMC
+## PMC
 
-The PMC (Project Management Committee) is the entity that controls and leads the whole Nebula Graph projects. 
+The PMC (Project Management Committee) is the entity that controls and leads the whole Nebula Graph projects.
 PMC members are responsible for voting new Maintainers or Committers and the authority to make all major decisions for Nebula Graph. See [PMC List](./pmc-list.md) for the list of PMC members.
 
-
-### Maintainer
+## Maintainer
 
 Maintainers are the planner and designer of the repository, with the right to merge branches into the master. The appointment is for one year. A Maintainer should:
 
 - Set technical directions, roadmap, and priorities for the repository
 - Drive the development forward and ensure newcomers, as well as long-time contributors, have a great experience
-- Ensure the overall quality of the repository
+- Make sure the overall quality of the repository
```

Comment:
it is not appropriate to replace "ensure" with "make sure" directly here.

Reply:
Pay attention to the overall quality of the repository